### DISCUSSION TITLE
ERROR - No share accepted in 30 minutes. Something is wrong, restart miner.

### DISCUSSION THREAD

**Discussion by BryanRice800** [NebuTech/NBMiner#703] [Ideas]
Hello

I don't really see this as an error, but given the message and the timings I see it the logs, is there a way to make this configurable?
Ideally I think it would be better if I could lower the threshold to 10 or 15 min to restart the miner if no shares have been accepted.

Thank you

**Comment by Tom5051** [comment]
you can configure this. 

-share-check 10
